From 7a4ecc5d55e2ca80730c8badef669d7d0366b203 Mon Sep 17 00:00:00 2001
From: Brock Noland <brock@apache.org>
Date: Fri, 6 Mar 2015 02:16:27 +0000
Subject: [PATCH 0069/1164] HIVE-9872 - Making yarn-cluster default for Hive
 on Spark [Spark Branch] (Chao via Brock)

git-svn-id: https://svn.apache.org/repos/asf/hive/branches/spark@1664541 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit c0d6a6e08907a0ad3f29af58235a4902e9276a1b)
---
 .../hive/ql/exec/spark/HiveSparkClientFactory.java |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ql/src/java/org/apache/hadoop/hive/ql/exec/spark/HiveSparkClientFactory.java b/ql/src/java/org/apache/hadoop/hive/ql/exec/spark/HiveSparkClientFactory.java
index 7d7e1a6..058d91d 100644
--- a/ql/src/java/org/apache/hadoop/hive/ql/exec/spark/HiveSparkClientFactory.java
+++ b/ql/src/java/org/apache/hadoop/hive/ql/exec/spark/HiveSparkClientFactory.java
@@ -46,7 +46,7 @@
   protected static final transient Log LOG = LogFactory.getLog(HiveSparkClientFactory.class);
 
   private static final String SPARK_DEFAULT_CONF_FILE = "spark-defaults.conf";
-  private static final String SPARK_DEFAULT_MASTER = "local";
+  private static final String SPARK_DEFAULT_MASTER = "yarn-cluster";
   private static final String SPARK_DEFAULT_APP_NAME = "Hive on Spark";
   private static final String SPARK_DEFAULT_SERIALIZER = "org.apache.spark.serializer.KryoSerializer";
   private static final String SPARK_DEFAULT_REFERENCE_TRACKING = "false";
-- 
1.7.9.5

